psqlgraph/mocks.py

```diff
@@ -412,9 +413,9 @@ def create_random_subgraph(self, label, max_depth=10, leaf_labels=None,
                 child_cls = edge_info['type']
 
                 child_node = self.node_factory.create(
-                    child_cls.label, all_props=all_props)
+                    child_cls.get_label(), all_props=all_props)
```

Comment:
probably better to use `label` here

Reply:
get_label() works too, so why not use it?

Reply:
mostly for consistency